### PR TITLE
Mm feature/update listitem singlelist styling

### LINKED ISSUE #44: Unify the styles for `ListItem` and `SingleList`
Use same overall styles, fonts on both pages

### PR DESCRIPTION
**Pull request by marshjaja**
## Description
- This PR implements the styles defined in the Figma files for the `Home` and `List` pages, along with all accompanying files.
   - _Slightly reduced the layout width for the navbar on desktop view to ensure the `NavBar` is not too spaced out._
 

## Related Issue

Close #44, #47 

## Acceptance Criteria
The acceptance criteria for this PR are specified in [Issue #14](https://github.com/the-collab-lab/tcl-79-smart-shopping-list/issues/14).

## Type of Changes

Use one or more labels to help your team understand the nature of the change(s) you’re proposing. E.g., `bug fix` or `enhancement` are common ones.
`enhancement`
`fix`

#### DESKTOP
Single List dark mode
Your lists dark mode
Your list Light mode
Single List Light mode
Screenshot 2024-10-10 at 15:08:02
Screenshot 2024-10-10 at 15:08:14

----
----

#### DESKTOP
Screenshot 2024-10-10 at 15:10:45
Screenshot 2024-10-10 at 15:10:58

#### MOBILE 
Screenshot 2024-10-10 at 14:55:03
Screenshot 2024-10-10 at 14:55:19
Screenshot 2024-10-10 at 14:55:39
Screenshot 2024-10-10 at 14:55:57
Screenshot 2024-10-10 at 15:03:01

## Testing Steps / QA Criteria

- Do a `git pull` and `git checkout mm-feature/update-listitem-singlelist-styling`
- On the `Home` and `List` pages you can see the styles implemented. You can toggle between Light mode and Dark mode to test the theme-switching functionality.

----
 Some styles might still need to be updated depending on the teams preference
 

 maybe adding some sort of background behind list to break out the white and black backgrounds.
 - examples (Not sure about the look):

Screenshot 2024-10-10 at 16:11:11
Screenshot 2024-10-10 at 16:11:44
Screenshot 2024-10-10 at 16:13:46

### CODE REVIEW

File: src/views/Layout.jsx [hunk 1]
```diff
@@ -26,7 +26,8 @@ export function Layout() {
 				<div className="Layout text-black dark:text-white bg-white dark:bg-black">
 					<NavBar darkMode={darkMode} toggleDarkMode={toggleDarkMode} />
 
-					<main className="mx-auto px-8 md:px-8 lg:px-20 w-full max-w-screen-sm min-h-screen flex flex-col">
+					<main className="mx-auto px-8 md:px-8 lg:px-20 w-full max-w-screen-sm min-h-screen flex flex-col rounded-xl pt-20">
```
Comment (Hudamabkhoot) [inline src/views/Layout.jsx]: Adding bottom padding here would be good so main isn't covered by the nav bar when you scroll to the bottom

**Review by Hudamabkhoot — CHANGES_REQUESTED** [review]
looks amazing well done! I left a comment for you but I couldn't find this other thing in this pull request to comment on it but i think removing the label from the search bars would look better!

**Review by joriordan332 — CHANGES_REQUESTED** [review]
This looks brilliant! The only small thing I noticed and it's only on really small devices is that if the item name is a little longer like raspberries for example then the delete button doesn't stay within the box.
Screenshot 2024-10-11 120831

**Review by Wyna-7 — APPROVED** [review]
It looks incredible, great job Marcia!
The only thing I would change is the position for the sign out and dark/light toggle buttons, I personally prefer the sign out one on the left. No biggie though!

**Comment by marshjaja** [comment]
@Wyna-7 I have swap them around on mobile it does look better i hope this is want your meant?

**Comment by marshjaja** [comment]
@joriordan332 very good spot this one Jonathan👏🏾, forgot to check!

**Comment by marshjaja** [comment]
@Hudamabkhoot  for the labels I can't remove them for accessibility reason. I have added button padding very good spot!🤾🏾‍♀️

**Review by joriordan332 — APPROVED** [review]

**Review by Hudamabkhoot — APPROVED** [review]

**Pull request merged**